tests/ssg_test_suite/rule.py

```diff
@@ -150,7 +152,24 @@ def _matches_target(rule_dir, targets):
         return False
 
 
-def _get_scenarios(rule_dir, scripts):
+def _matches_platform(scenario_platforms, benchmark_cpes):
+    if "multi_platform_all" in scenario_platforms:
+        return True
+    scenario_cpes = set()
+    for platform in scenario_platforms:
+        try:
+            if platform.startswith("multi_platform_"):
+                platform_cpes = set(common.TEST_MULTI_PLATFORMS[platform])
+            else:
+                platform_cpes = set(common.TEST_PLATFORMS[platform])
+        except KeyError:
+            raise ValueError(
+                "Platform '%s' is not supported by the test suite." % platform)
```

Comment:
What is the reason a platform is not supported for testing?
Is it because there wouldn't be a mapping from the product name to its cpe?

Reply:
Yes, it's exactly that. I agree it's kind of stupid. It is related to previous comment. Basically the problem is that it isn't easy to get such a mapping now and we'll need to change the way we define product names and CPEs to prevent maintaining a separate list of mappings for the test suite.